### DISCUSSION TITLE
申请退款！！反馈好几次了

### DISCUSSION THREAD

**Discussion by pplive98** [getlantern/lantern-forum-cn#1011] [General]
买了专业版，连个网站都打不开，已经反馈好几次了，连个正常的售后都做不到？请退款！账号274892121@qq.com

**Comment by MaxMKMax** [comment]
@pplive98 您好. 我不负责管理support邮箱. 我看不到您的反馈. 请您提供我支付单据, 我马上给您提交问题上去处理.
